app/repositories/[id]/page.tsx

```diff
@@ -1,13 +1,14 @@
 'use client'
-import { Suspense } from 'react'
+import { Suspense, use } from 'react';
 import { RepositoryViewPageContent } from './page-content'
 import { RedirectFromHandleToDid } from '@/repositories/RedirectFromhandleToDid'
 
-export default function RepositoryViewPage({
-  params,
-}: {
-  params: { id: string }
-}) {
+export default function RepositoryViewPage(
+  props: {
+    params: Promise<{ id: string }>
+  }
+) {
+  const params = use(props.params);
```

Comment:
Is the promise here a next 15 thing?

Reply:
yep https://nextjs.org/docs/app/building-your-application/routing/dynamic-routes#example